### DISCUSSION TITLE
"createElementVNode" is imported from external module...

### DISCUSSION THREAD

**Discussion by rivatove** [tailwindlabs/heroicons#844] [Help]
Hey, I'm getting this annoying warning during the execution of "vite build":

"createElementVNode" is imported from external module "vue" but never used in... (long list of project files)

Using the newest version of Vue library along with Vite.

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

Can you create a minimal reproduction repo and share it as well so that we can take a look?
